crates/host/src/wasmbus/mod.rs

```diff
@@ -3727,6 +3780,30 @@ fn ensure_actor_capability(
     Ok(())
 }
 
+fn injector_to_headers(injector: &OtelHeaderInjector) -> HeaderMap {
+    injector
+        .as_ref()
+        .iter()
+        .filter_map(|(k, v)| {
+            // There's not really anything we can do about headers that don't parse
+            let name = HeaderName::from_str(k.as_str()).ok();
+            let value = HeaderValue::from_str(v.as_str()).ok();
```

Comment:
same as above, I'm fine with `use async_nats::header::{HeaderName, HeaderValue}` inside the function if you really want it, but I'd rather have all of these types spelled out completely